Performance.md

```diff
@@ -203,6 +293,16 @@ Alternatively, you can also enable the `skipLibCheck` flag to skip checking *all
 
 These two options can often hide misconfiguration and conflicts in `.d.ts` files, so we suggest using them *only* for faster builds.
 
+## Using Faster Variance Checks
+
+Is a list of dogs a list of animals?
+That is, is `List<Dog>` assignable to `List<Animals>`?
+The straightforward way to find out is to do a structural comparison of the types, member by member.
+Unfortunately, this can be very expensive.
+However, if we know enough about `List<T>`, we can reduce this assignability check to determining whether `Dog` is assignable to `Animal` (i.e. without considering each member of `List<T>`).
+(In particular, we need to know the [variance](https://en.wikipedia.org/wiki/Covariance_and_contravariance_(computer_science)) of the type parameter `T`.)
+The compiler can only take full advantage of this potential speedup if the `strictFunctionTypes` flag is enabled (otherwise, it uses the slower, but more lenient, structural check).
```

Comment:
It's unclear what the action item is for this one at a glance. Maybe add

```md
For this reason, we recommend building with `--strictFunctionTypes` (which is enabled by default under `--strict`).
```

Reply:
Not sure why this one wasn't a suggested edit. 😛